src/interpreter/ByteCode.h

```diff
@@ -47,6 +47,7 @@ class ByteCode {
     }
 #endif
 
+    size_t getSize();
```

Comment:
There are `virtual size_t byteCodeSize()` functions below, they should be deleted.

Reply:
You are right, they are unused now, I deleted them.